app/components/welcome-page-copy.css

```diff
@@ -35,11 +30,11 @@ h1 {
 
 a:link, .guide:link,
 a:visited, .guide:visited {
-  color: ember-orange;
```

Comment:
This value was defined at the top of the file with: `@value ember-orange from 'css-modules-to-scoped-css/styles/app';`. This feature is specific to ember-css-modules. 

Instead, I use `var(--ember-orange);` to reference a CSS variable that has been defined globally in `app.css`.